src/common/hooks/useRestoreToken.tsx

```diff
@@ -44,34 +44,66 @@ export const useRestoreToken = (
   const [errorMessage, setErrorMessage] = useState<string>();
   const [state, setState] = useState<ContractFunctionState>("UNINITIALIZED");
 
+  const sendToTitleEscrow = async (previousBeneficiary: string, previousHolder: string): Promise<void> => {
+    if (!tokenId) throw new Error("Ownership data is not provided");
+    if (!contractInstance?.address) throw new Error("Token Registry Instance should have address");
+    setState("PENDING_CONFIRMATION");
```

Comment:
`PENDING_CONFIRMATION` refers to the state where the transaction is constructed, signed, and sent to a node. It is waiting to be mined and confirmed. At this point in the code, the transaction has not been constructed...